llvm-runtimes/compiler-rt/compiler-rt-22.0.0.9999.ebuild

```diff
@@ -63,8 +63,6 @@ test_compiler() {
 }
 
 src_configure() {
-	llvm_prepend_path "${LLVM_MAJOR}"
```

Comment:
I'm sorry if I wasn't clear but I think `llvm_prepend_path` is still needed, as with `use clang || use test` you need clang, and that one needs to come from `BROOT`.

Reply:
Should it be `llvm_prepend_path -b` then, to make sure clang is taken from `BROOT`?

If we continue using `llvm_prenend_path` without flags, then we still have the original problem I'm trying to fix - a `cross-*` package is going to keep a foreign clang binary from the sysroot (`/usr/aarch64-unknown-linux-musl/usr/lib/llvm/19/bin/aarch64-unknown-linux-musl-clang`), instead of a clang wrapper from the BROOT (`/usr/lib/llvm/19/bin/aarch64-unknown-linux-musl-clang`).

Reply:
We resolved in on IRC.